tests/sanity/ignore-2.10.txt

```diff
@@ -9,4 +9,8 @@ molecule/default/roles/helm/files/appversionless-chart/templates/configmap.yaml
 molecule/default/roles/helm/files/test-chart-v2/templates/configmap.yaml yamllint!skip
 molecule/default/roles/helm/files/test-chart/templates/configmap.yaml yamllint!skip
 plugins/module_utils/k8sdynamicclient.py import-2.7!skip
-plugins/module_utils/k8sdynamicclient.py import-3.7!skip
```

Comment:
I'm surprised with cannot import with Python 3.7. Do we use some python 3.8+ features?

Reply:
I think this is needed because of the bare imports in the new files. We had to do the same thing previously for the k8sdynamicclient.py.